### PR TITLE
swarm/network: remove spammy log line

### PR DESCRIPTION
**Pull request by nonsense**

### CODE REVIEW

**Review by janos — APPROVED** [review]

File: swarm/network/stream/delivery.go [hunk 1]
```diff
@@ -255,7 +255,7 @@ func (d *Delivery) RequestFromPeers(ctx context.Context, req *network.Request) (
 			}
 			sp = d.getPeer(id)
 			if sp == nil {
-				log.Warn("Delivery.RequestFromPeers: peer not found", "id", id)
+				//log.Warn("Delivery.RequestFromPeers: peer not found", "id", id)
```
Comment (zelig) [inline swarm/network/stream/delivery.go:258]: lets leave this. this warning occurring too many times is indicating a bug.

**Review by zelig — CHANGES_REQUESTED** [review]

File: swarm/network/stream/delivery.go (hunk 1 above)
Comment (nonsense) [reply swarm/network/stream/delivery.go:258]: I understand that this is a bug, but this is a very spammy log line, and it is not helpful.

screen shot 2018-11-21 at 12 48 53

400k times for a single peer. Users are complaining that this is filling their disks with logs, and rightly so.

---

I am currently looking into it, but we should not have another release with this spammy log.

The peer in question in the logs is actually present in the peer list, so I am not sure why `d.getPeer(id)` is returning nil.

**Review by acud — APPROVED** [review]
lgtm with leaving the commented line in, let's follow up on this in a separate issue

File: swarm/network/stream/delivery.go (hunk 1 above)
Comment (holisticode) [reply swarm/network/stream/delivery.go:258]: If it's a bug and we just silently "ignore" it, then we should open an issue to investigate it and find a solution.

**Review by holisticode — APPROVED** [review]

**Comment by nonsense** [comment]
@holisticode here is the issue. https://github.com/ethersphere/go-ethereum/issues/999

**Review by zelig — APPROVED** [review]

**Comment by zelig** [comment]
@nonsense please use status labels

**Comment by nonsense** [comment]
I am not sure what status label you are talking about? This PR has been open for days with responses to your comments, and because of this there was no action on it. Also you had `Changes requested` status on it, so I am not sure what I was supposed to do it that case.

**Comment by nonsense** [comment]
Closing this as we are piggy-backing it to another PR.

**Pull request closed without merging**